integration-tests/resources/executors/gradle/.gitattributes

```diff
@@ -0,0 +1,9 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Linux start script should use lf
+/gradlew        text eol=lf
+
+# These are Windows script files and should use crlf
+*.bat           text eol=crlf
+*.cmd           text eol=crlf
```

Comment:
I'm anxious to ask what horrible issues with gradle forced this.

Reply:
gradle and mvn have wrappers that are helpful to stabilize the build, bu default they come with windows support, and bat/cmd uses CRLF, it's a git problem mostly